gulpfile.js

```diff
@@ -77,150 +74,143 @@ gulp.task('serve', function () {
  * More about Nunjucks here:
  * https://mozilla.github.io/nunjucks/
  */
-gulp.task('eleventy', function (cb) {
-  cp.exec('npx eleventy', function (err, stdout, stderr) {
+function eleventy(callback) {
+  cp.exec("npx eleventy", function(stdout, stderr) {
     console.log(stdout);
     console.log(stderr);
-    cb(err);
-  })
-})
+    callback();
+  });
+}
 
-gulp.task('eleventy:watch', function () {
-  const eleventy = cp.spawn('npx', ['eleventy', '--watch']);
+function watchEleventy() {
+  const eleventy = cp.spawn("npx", ["eleventy", "--watch"]);
 
-  const eleventyLogger = function (buffer) {
-    buffer.toString()
+  const eleventyLogger = function(buffer) {
+    buffer
+      .toString()
       .split(/\n/)
-      .forEach((message) => gutil.log('Eleventy: ' + message));
+      .forEach(message => gutil.log("Eleventy: " + message));
   };
+  eleventy.stdout.on("data", eleventyLogger);
+  eleventy.stderr.on("data", eleventyLogger);
+}
 
-  eleventy.stdout.on('data', eleventyLogger);
-  eleventy.stderr.on('data', eleventyLogger);
-})
+function compileSass() {
+  return src("src/sass/**/*.scss")
+    .pipe(sass({ outputStyle: "expanded" }).on("error", sass.logError))
+    .pipe(dest("docs/css/"));
+}
 
-gulp.task('sass', function () {
-  return gulp
-    .src('src/sass/**/*.scss')
-    .pipe(sass({ outputStyle: 'expanded' }).on('error', sass.logError))
-    .pipe(gulp.dest('docs/css/'));
-});
-
-gulp.task('sass:watch', function() {
-  gulp.watch('src/sass/**/*.scss', ['sass']);
-});
+function watchSass() {
+  watch("src/sass/**/*.scss", compileSass);
+}
 
 /**
  * Uses Rollup to compile ES6 down to browser JS with a UMD wrapper.
  * See more here:
  * https://rollupjs.org/guide/en#gulp
  */
-gulp.task('js', () => {
+function compileJS() {
```

Comment:
This probably needs re-worked a little bit. Looks like you still running it through the rollup `babel()` plugin, but if you look at the compiled JS file inside of the `docs/` folder, it is not being transpiled down to ES5 (browser-safe). E.g. there are still `const` declarations, and arrow functions.

Reply:
I can confirm on this one that the JS isn't being transpiled to ES5 when you build the `dist/` folder either. E.g. `npm run build`